runtime/near-runtime-fees/src/lib.rs

```diff
@@ -63,6 +71,10 @@ pub struct RuntimeFeesConfig {
 
     /// Pessimistic gas price inflation ratio.
     pub pessimistic_gas_price_inflation_ratio: Rational,
+
+    /// New EVM deposit.
+    #[serde(with = "u128_dec_format", default = "default_evm_deposit")]
+    pub evm_deposit: Balance,
```

Comment:
what is this used for

Reply:
Amount of $NEAR to create new EVM vault.